src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star__stars {
+  background-image: url(./images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
The selectors for `.stars--1 :nth-child(1)`, `.stars--2 :nth-child(-n + 2)`, etc., have a space before `:nth-child`, which increases specificity and may cause unexpected behavior. Consider removing the space to ensure the pseudo-class applies directly to the children of the `.stars--X` elements.